website/content/docs/connect/config-entries/service-resolver.mdx

```diff
@@ -476,6 +526,16 @@ spec:
           description:
             'Specifies the destination datacenter to resolve the service from.',
         },
+        {
+          name: 'Peer',
+          type: 'string: ""',
+          description:
+            `Specifies the destination cluster peer to resolve the service
+                from. When peer is specified, the failover target relies on Envoy's
+                outlier detection for health checks. Consul relies on outlier
+                detection because imported service health checks don't account
```

Comment:
```suggestion
                from. When the peer is specified, the failover target relies on Envoy's
                outlier detection for health checks. Consul relies on outlier
                detection because imported service health checks do not account
```
No contractions per the style guide